messages/teleporter/message_manager_test.go

```diff
@@ -49,6 +49,17 @@ var (
 	}
 )
 
+func getChainIDBytes(id string) ([32]byte, error) {
+	chainID, err := ids.FromString(id)
```

Comment:
If I'm reading this helper correctly, `decodeChainID` may be a more descriptive name. 

Also, I think we should be able to have the implementation be just
```
return ids.FromString(id)
```

because `ids.ID` is an alias for `[32]byte` already. If that's the case, it may be easiest to remove it all together since it's already a single function call.

Reply:
fixed